launcher/modplatform/packwiz/Packwiz.cpp

```diff
@@ -346,4 +346,26 @@ auto V1::getIndexForMod(const QDir& index_dir, QVariant& mod_id) -> Mod
     return {};
 }
 
+// return false if a > b
+bool V1::versionsSort(QString a, QString b)
+{
+    QStringList splitA = a.split('.');
+    QStringList splitB = b.split('.');
+    int i = 0;
+    for (i; i < std::min(splitA.size(), splitB.size()); i++) {
+        int intA = splitA[i].toInt();
+        int intB = splitB[i].toInt();
+        if (intA == intB) {
+            continue;
+        } else {
+            return intA > intB;
+        }
+    }
+    if (splitA.size() == splitB.size()) {
+        return ((splitA[i - 1].size() <= splitB[i - 1].size()));
+    }
+    return splitA.size() > splitB.size();
+
+}
```

Comment:
I don't think you need so much code
```suggestion
static bool versionsSort(QString a, QString b)
{
    return Version(a) <= Version(b);
}
```

Reply:
yeah, we have a solid version parser. best to use it